.github/workflows/pr_basic_compilation_check.yml

```diff
@@ -0,0 +1,30 @@
+name: Basic Compilation Check
+
+on: [push, pull_request]
+
+jobs:
+  basic_complication_check:
+    # Run all steps in the compilation testing containers
+    container: runetest/compilation-testing:anolis8.4
+
+    # Use GitHub-hosted runner Ubuntu 20.04
+    runs-on: ubuntu-20.04
+
+    defaults:
+      run:
+        shell: bash
+        working-directory: ${{ github.workspace }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Compile "rats-tls"
```

Comment:
please split different build modes into different steps to clarify the steps. for example: 

```sh
- name: Compile "rats-tls" host mode
- name: Compile "rats-tls" occlum mode
- name: Compile "rats-tls" sgx mode
```